src/main/kotlin/org/neo4j/graphql/handler/projection/ProjectionBase.kt

```diff
@@ -230,7 +242,7 @@ open class ProjectionBase {
             .filterIsInstance<Field>()
             .map {
                 val value = when (it.name) {
-                    NEO4j_FORMATTED_PROPERTY_KEY -> "$variable.${field.name}"
+                    NEO4j_FORMATTED_PROPERTY_KEY -> "toString($variable.${field.name})"
```

Comment:
why did we have to do this?

Reply:
This was a Bug I discovered by comparing to the js-lbrary. In case we should return the `formatted` Date/DateTime/* property, this ensures, that a string will be returned instead the Date/DateTime/*-Object.